scss/_patterns_matrix.scss

```diff
@@ -121,7 +122,16 @@
 
   .p-matrix__title {
     @extend %vf-heading-4;
-    @extend %u-no-margin--bottom--h4;
+
+    @media (min-width: $breakpoint-heading-threshold) {
+      margin-bottom: -#{map-get($nudges, nudge--h4)} !important;
```

Comment:
Why is `!important` needed here? Class name styling should give enough specificity to override h* styling.

This applies to line 132 as well.